Makefile

```diff
@@ -5,8 +5,8 @@
 VERSION ?=latest
 # Default image tag
 
-SIDECAR_IMG ?= quay.io/aneeshkp/cloud-event-proxy:$(VERSION)
-CONSUMER_IMG ?= quay.io/aneeshkp/cloud-native-event-consumer:$(VERSION)
+IMG ?= quay.io/redhat-cne/cloud-event-proxy:$(VERSION)
```

Comment:
Use openshift image? `quay.io/openshift/origin-cloud-event-proxy`

Reply:
this for developer, you can't build and push it to this locally

Reply:
alright changed it to openshift, anyway user can override it with $IMG